docs/codeql/codeql-cli/publishing-and-using-codeql-packs.rst

```diff
@@ -0,0 +1,74 @@
+.. publishing-and-using-codeql-packs:
+
+Publishing and using CodeQL packs
+=================================
+
+You can publish your own CodeQL packs and use packs published by other people.
+
+.. pull-quote::
+
+   Note
+
+   The CodeQL package manager is currently in beta and subject to change. During the beta, CodeQL packs are available only in the GitHub Package Registry (GHPR). You must use version 2.5.8 or later of the CodeQL CLI to use the CodeQL package manager.
+
+Configuring the ``qlpack.yml`` file before publishing
+-----------------------------------------------------
+
+You can check and modify the configuration details of your CodeQL pack prior to publishing. Open the ``qlpack.yml`` file in your preferred text editor.
+
+   library: # set to true if the pack is a library. Set to false or omit for a query pack
+   name: <scope>/<pack>
+   version: x.x.x
+   description: 
+   default-suite: # a query-suite file that has been inlined
+       - query:
+   default-suite-file: default-queries.qls # a pointer to a query-suite in this pack
+   license:
+   dependencies:
+
+- ``name:`` must follow the <scope>/<pack> format, where <scope> is the GitHub organization that you will publish to and <pack> is the name for the pack.
+- Only one of ``default-suite`` or ``default-suite-file`` is allowed. Both options define a default query suite to be run.
+
+Running ``codeql pack publish``
```

Comment:
Maybe call out somewhere that we currently only support GHCR.

Reply:
👍
https://github.com/github/codeql/pull/6339/files/87005b93e3511ae4fcfb654140fe90ec03ecf3ef#diff-4b695930f49b7fbf3ecaf6911a3f9853648dc914011209e2488a307af9a3d96cR12